myapp/app/models/task.rb

```diff
@@ -18,7 +19,9 @@ class Task < ApplicationRecord
 
   # スコープ
   scope :where_title, -> (title) { where('title like ?', "%#{title}%") if title.present? }
+  scope :where_label, -> (label) { where('labels.id = ?', "#{label}") if label.present? }
```

Comment:
引数はidのはずですが、これではオブジェクトを受け取るように見えます。
また、異なるモデルの検索をするときは以下のように書くと良いです。
```suggestion
  scope :where_label, -> (label_id) { where(labels: { id: label_id }) if label_id.present? }
```

Reply:
ありがとうございます！修正しました。